infrastructure/code/convert.go

```diff
@@ -391,6 +391,21 @@ func (s *SarifConverter) toIssues(baseDir string) (issues []snyk.Issue, err erro
 				CWEs:                testRule.Properties.Cwe,
 			}
 
+			// this can be an array of multiple suppressions in SARIF
+			// but we only store one ignore for now
```

Comment:
Why?

Reply:
There can only be one ignore per finding, as far as I know. I'm not sure what would happen though if an ignore expires and the user wants to "renew" it - do they create another on top of it but only one is "active" or would it just get replaced. Maybe I should ask

Reply:
https://snyk.slack.com/archives/C06MZKWG0NB/p1710328751971279?thread_ts=1710150868.607839&cid=C06MZKWG0NB